src/com/lftechnology/java/training/dipak/employeemanagement/controller/EmployeeController.java

```diff
@@ -0,0 +1,233 @@
+
+package com.lftechnology.java.training.dipak.employeemanagement.controller;
+
+import java.io.Console;
+import java.sql.ResultSet;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.dipak.employeemanagement.domain.Employee;
+import com.lftechnology.java.training.dipak.employeemanagement.domain.UserType;
+import com.lftechnology.java.training.dipak.employeemanagement.service.EmployeeService;
+import com.lftechnology.java.training.dipak.employeemanagement.service.ServiceFactory;
+
+/**
+ * <p> This class is used to take the data from the user based on the various
+ * functionalities provided by its member function.</p>
+ * 
+ * @author Dipak Thapa<dipakthapa@lftechnology.com>
+ */
+public class EmployeeController {
+
+	private static final Logger LOGGER =
+		Logger.getLogger(EmployeeController.class.getName());
+
+	/**
+	 * This method takes the input from the console and sets the employee object
+	 * and calls the addEmployee method in service layer.
+	 * 
+	 * @param e
+	 * @param sc
+	 * @return count
+	 */
+	public int addEmployee(Employee e, Scanner sc) {
+
+		String userName = null;
+		String password = null;
+		String fullName = null;
+		String department = null;
+		String address = null;
+		int count = 0;
+		boolean isTerminated = false;
+		UserType role = null;
+		try {
+			for (;;) {
+				try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this nested try block into a separate method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1141)